lib/checkUrl.js

```diff
@@ -0,0 +1,12 @@
+const axios = require("axios");
+
+async function checkUrl(url) {
+    try {
+        const res = await axios.get(url);
+        return true;
```

Comment:
it can only check whether the URL returns `200`
In fact, it's possible it returns `200` but the content is not you wanted.
You may need to check the content has the thing.